package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "eslint-plugin-yola",
+  "version": "0.1.0",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js ./**/*.js",
+    "test": "jest",
+    "prepublish": "npm run lint && npm run test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yola/eslint-plugin-yola.git"
+  },
+  "author": "Yola Engineering <engineers@yola.com>",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/yola/eslint-plugin-yola/issues"
+  },
+  "homepage": "https://github.com/yola/eslint-plugin-yola#readme",
+  "dependencies": {
+    "babel-eslint": "8.x",
+    "eslint": "4.x",
+    "eslint-config-airbnb": "16.x",
+    "eslint-config-airbnb-base": "12.x",
+    "eslint-plugin-import": "2.x",
+    "eslint-plugin-jsx-a11y": "6.x",
+    "eslint-plugin-react": "7.x",
+    "lodash": "^4.17.4"
+  },
+  "devDependencies": {
+    "eslint": "4.x",
+    "jest": "^21.2.1"
+  },
+  "peerDependencies": {
+    "eslint": "4.x"
+  },
+  "jest": {
```

Comment:
Oh, something new. I've never used jest before, any particular reason why you prefer it?

Reply:
The snapshot feature makes whole config tested in two lines literally

Reply:
It also tends to be faster than mocha. I've started using it after @bebraw 's talk in Kharkiv last year.

Reply:
You can't reference a talk but not provide a link to watch!

Reply:
Sure. I just was on a bus in a middle of nowhere writing that https://www.youtube.com/watch?v=1niBVCLfAGI